pandas/core/groupby/ops.py

```diff
@@ -96,6 +96,7 @@
 )
 
 if TYPE_CHECKING:
+    from pandas.core.arrays.string_ import StringDtype
```

Comment:
the annotation that uses this may no longer be accurate?

Reply:
Good point. Added to `| ArrowDtype` to the annotation where this is used